components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -237,22 +221,136 @@ function OrgSettingsForm(props: { org?: OrganizationInfo }) {
                 disabled={isLoading || !org?.isOwner}
             />
 
-            <Heading2 className="pt-12">Workspace Settings</Heading2>
-            <TextInputField
-                label="Default Image"
-                // TODO: Provide document links
-                hint="Use any official Gitpod Docker image, or Docker image reference"
-                placeholder={globalDefaultImage}
-                value={defaultWorkspaceImage}
-                onChange={setDefaultWorkspaceImage}
-                disabled={isLoading || !org?.isOwner}
+            <Heading2 className="pt-12">Workspace Images</Heading2>
+            <Subheading className="max-w-2xl">
+                Choose which workspace images you want to use for your workspaces.
+            </Subheading>
+
+            <WorkspaceImageButton
+                disabled={!org?.isOwner}
+                image={settings?.defaultWorkspaceImage ?? globalDefaultImage ?? ""}
+                onClick={() => setShowImageEditModal(true)}
             />
 
-            {org?.isOwner && (
-                <Button htmlType="submit" className="mt-4" disabled={!org.isOwner}>
-                    Update Default Image
-                </Button>
+            {showImageEditModal && (
+                <OrgDefaultWorkspaceImageModal
+                    settings={settings}
+                    globalDefaultImage={globalDefaultImage}
+                    onClose={() => setShowImageEditModal(false)}
+                />
             )}
         </form>
     );
 }
+
+function WorkspaceImageButton(props: { image: string; onClick: () => void; disabled?: boolean }) {
+    function parseDockerImage(image: string) {
+        // https://docs.docker.com/registry/spec/api/
+        let registry, repository, tag;
+        let parts = image.split("/");
+
+        if (parts.length > 1 && parts[0].includes(".")) {
+            registry = parts.shift();
+        } else {
+            registry = "docker.io";
+        }
+
+        const remaining = parts.join("/");
+        [repository, tag] = remaining.split(":");
+        if (!tag) {
+            tag = "latest";
+        }
+        return {
+            registry,
+            repository,
+            tag,
+        };
+    }
+
+    return (
+        <InputField disabled={props.disabled} className="w-full max-w-lg">
+            <div className="flex items-center bg-gray-50 dark:bg-gray-800 p-3 justify-between rounded-lg">
+                <div className="flex items-center overflow-hidden" title={props.image}>
+                    <span className="w-5 h-5 mr-1">
+                        <Stack />
+                    </span>
+                    <span className="truncate">{parseDockerImage(props.image).repository}</span>
+                    &nbsp;&middot;&nbsp;
+                    <span className="flex-none w-16 truncate text-gray-500 dark:text-gray-500">
```

Comment:
**thought**: Regarding width, ideally if we could split the remaining area. outside the _Change_ link button into two, we would have a max width of 50% of the remaining space. I'm sure with flexbox we could achieve something like this but it's really minor. Let's leave this as is until we run into any issues.